src/pvr2wmc.cpp

```diff
@@ -1142,6 +1142,9 @@ PVR_ERROR Pvr2Wmc::GetRecordings(ADDON_HANDLE handle)
 			xRec.iEpgEventId = atoi(v[18].c_str());
 		}
 
+		/* TODO: PVR API 5.0.0: Implement this */
+		xRec.iChannelUid = PVR_CHANNEL_INVALID_UID;
```

Comment:
This field should already be available in the response message at array position 17

So this could be replaced with the following:

```
// Kodi PVR API 5.0.0 adds Channel ID field
if (v.size() > 18)
{
    xRec.iEpgEventId = atoi(v[17].c_str()));
}
```